### PR TITLE
Updated view, routes, and controller

### PR DESCRIPTION
**Pull request by leel8on**
Updates view, routes, and controller to allow for selection of type of deck

### CODE REVIEW

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ public Game(){
 
 
 
-	public abstract void builDeck();		//make the class abstract - to be implemented in the child classes
+	public abstract void buildDeck();		//make the class abstract - to be implemented in the child classes
```
Comment (athai) [inline src/main/java/models/Game.java:26]: Good catch!

File: src/main/java/views/AcesUp/SPacesUp.flt.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+            <tr><td class="cardLocation l2"></td></tr>
+            <tr><td class="cardLocation l3"></td></tr>
+            <tr><td class="cardLocation l4"></td></tr>
+            <tr><td class="cardLocation l5"></td></tr>
+            <tr><td class="cardLocation l6"></td></tr>
+            <tr><td class="cardLocation l7"></td></tr>
+            <tr><td class="cardLocation l8"></td></tr>
+            <tr><td class="cardLocation l9"></td></tr>
+            <tr><td class="cardLocation l10"></td></tr>
+            <tr><td class="cardLocation l11"></td></tr>
```
Comment (athai) [inline src/main/java/views/AcesUp/SPacesUp.flt.html]: The Spanish deck only has 40 cards, so you can remove/comment out rows 11-13 for each column.

**Comment by athai** [comment]
We never fixed the error button. So, if you could comment out the section where it's set to true (I believe it's in Game.java: on the remove function), that'd be great.

**Comment by leel8on** [comment]
@athai I removed the extra rows and commented out where the error is set. I also grabbed the tests from your PR, but I can remove them if you want to add them in a separate PR

**Comment by athai** [comment]
Nah. If the tests pass and everything works then I say it's good! 

The Travis failures can be left for another day...

**Comment by athai** [comment]
Gonna merge so our main repo isn't completely broken anymore!

Lee is the real MVP. Travis... not so much.

**Pull request merged**